more/free-programming-playgrounds.md

```diff
@@ -266,6 +267,10 @@
 * [Scastie](https://scastie.scala-lang.org)
 
 
+### Scratch
+
+* [Scratch.mit.edu](https://scratch.mit.edu/create)
```

Comment:
In my opinion, this is up to you. The online playground itself is at https://scratch.mit.edu/create (this is what this section "playgrounds" suggests from my point of view), but it could be useful to link the entire domain to ease the information about Scratch and avoid dependences on /create that may change at some point.